dces-drc-integration/build.gradle

```diff
@@ -1,36 +1,38 @@
+import org.springframework.boot.gradle.plugin.SpringBootPlugin
+
 plugins {
 	id 'java'
 	id 'jacoco'
-	id 'org.springframework.boot' version '3.2.4'
-	id 'io.spring.dependency-management' version '1.1.4'
-	id "org.sonarqube" version "4.0.0.2929"
+	id 'org.springframework.boot' version '3.3.2'
+	id 'io.spring.dependency-management' version '1.1.6'
+	id "org.sonarqube" version "5.1.0.4882"
 }
 
 def versions = [
-		pitestVersion          : '1.4.10',
-		sentryVersion          : '6.15.0',
-		springdocVersion       : '2.3.0',
-		okhttpVersion          : '4.9.3',
+		pitestVersion          : '1.16.1',
+		sentryVersion          : '7.13.0',
+		springdocVersion       : '2.1.0',
+		okhttpVersion          : '4.12.0',
 		mockwebserverVersion   : '4.12.0',
 		reactorTestVersion     : '3.2.3',
-		springSecurityConfig   : '6.2.0',
-		springSecurityWeb      : '6.1.7',
-		springOAuthJose        : '6.1.1',
-		springOAuthClient      : '3.1.1',
-		springOAuthResource    : '3.1.1',
+		springSecurityConfig   : '6.3.1',
+		springSecurityWeb      : '6.3.1',
+		springOAuthJose        : '6.3.1',
+		springOAuthClient      : '3.3.2',
+		springOAuthResource    : '3.3.2',
 		springauthserverVersion: '1.0.1',
 		jjwtVersion            : '0.11.5',
-		jaxbXjc                : '4.0.2',
-		jaxbImpl               : '4.0.2',
-		jaxbCore               : '4.0.2',
+		jaxbXjc                : '4.0.5',
+		jaxbImpl               : '4.0.5',
+		jaxbCore               : '4.0.5',
 		jakartaActivation      : '2.0.1',
-		jakartaXmlBindApi      : '4.0.0',
-		glassfishJaxb          : '4.0.2',
-		resilience4jVersion    : '2.0.2',
-		wiremockVersion        : '2.35.1',
+		jakartaXmlBindApi      : '4.0.2',
+		glassfishJaxb          : '4.0.5',
+		resilience4jVersion    : '2.2.0',
+		wiremockVersion        : '3.0.1',
 		notifyVersion          : '4.1.0-RELEASE',
-		micrometerio           : '1.1.3',
-		prometheus           : '1.11.2',
+		micrometerio           : '1.3.2',
+		prometheus             : '1.13.2',
 		logbackEncoderVersion: '7.2'
```

Comment:
Also, to fix https://app.snyk.io/org/legal-aid-agency/project/823895b5-eecd-470c-8b29-d2979de9733b would be worth updating logback-stash-encoder to version 7.4 if possible